src/config.rs

```diff
@@ -122,7 +123,7 @@ pub struct Config {
     pub network: Network,
     pub db_path: PathBuf,
     pub daemon_dir: PathBuf,
-    pub daemon_cookie_file: PathBuf,
+    pub daemon_auth: Auth,
```

Comment:
Make sure it's not dumped into the log.

Reply:
https://github.com/romanz/electrs/pull/442/commits/ad46aa37ca9fc7d88f55226cbf3c4c07917fe24b